Script for synchronising SSM Parameters between environments

petter-a opened the pull request:
A script to help synchronise sandbox with the SSM Parameters defined in another environment:

This script extracts AWS SSM Parameters from an AWS account and
creates a file with the commands for recreating them in another.
Before running, make sure that you are logged on to the account
of which parameters you want to extract.

The currently active source account is "mitt-hbg-dev"

Type C to continue: C

The extraction has finished. Logon to the destination account using aws-vault exec <account>
and issue the commands from the file /tmp/parameters.txt to complete the copy.

yo-l1982 reviewed (commented):
Looks fine.

I think jq is rarely installed by default. Maybe a check in script or some requirement heads up?

Another thing about the ssm parameters ex. "navetEnvs" contains a bucket name and as they are globally unique they will need to be changed on the target.

petter-a commented:
> Looks fine.
> 
> I think jq is rarely installed by default. Maybe a check in script or some requirement heads up?
> 
> Another thing about the ssm parameters ex. "navetEnvs" contains a bucket name and as they are globally unique they will need to be changed on the target.

Moved to Bash and added additional checks for command availability. The uniqueness of the actual content of the ssm parameters cannot be handled by these scripts. It needs to be modified manually. (Scripts are essentially doing a copy, not considering what content is copied)
